cli/tests/services/register_cleanup.sh

```diff
@@ -5,7 +5,7 @@ set -euo pipefail
 export PC_SOCKET_PATH="${_FLOX_SERVICES_SOCKET}"
 
 function cleanup() {
-    echo "Shutting down process-compose"
+    echo "Shutting down process-compose" >&2
     # Need to allow for errors in the shutdown process:
     # https://github.com/F1bonacc1/process-compose/issues/197
     "$PROCESS_COMPOSE_BIN" down || true
```

Comment:
Can we remove this now?

Reply:
Please god just don't touch anything until after the release

Reply:
I'm a teeny-tiny bit nervous that we won't know whether it's properly working until we remove this because we're essentially bypassing the work of the watchdog for all of the tests.

Reply:
The issue is that running things programmatically gets you into situations that aren't how a user would use them, like the bug I just solved where `expect` wasn't cleaning up the shell process. In normal use when you close a shell it actually terminates and the terminal emulator cleans it up, so I think my preference boils down to:
- We have a deadline and can't afford to block all of CI on non-interactive edge cases
- Tests need to reflect reality, and I'm not convinced all of our tests do